### PR TITLE
[13.0][IMP] account_invoice_mgmt_atm: adds journal atm to res.users

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds journal atm to res.users

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod Enviroment test updated can u test it?

File: account_invoice_mgmt_atm/views/res_users_views.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_res_users_form_ATM_config" model="ir.ui.view">
+        <field name="name">res.users (ATM)</field>
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='messaging']" position="after">
+                <group name="atm_config" string="ATM Configuration">
```
Comment (dalonsod) [inline account_invoice_mgmt_atm/views/res_users_views.xml]: This data should be only accessed by a user with permissions in Accounting features (eg. see https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/security/ir.model.access.csv#L37)

File: account_invoice_mgmt_atm/controllers/controller.py [hunk 2]
```diff
@@ -79,22 +80,22 @@ def cajero_emision_completada_reciclaje(self, *args, **post):
...
-                payment_method_id = res_company_id.sudo().journal_ATM_id.outbound_payment_method_ids[0]
+                payment_method_id = user_id.sudo().journal_ATM_id.outbound_payment_method_ids[0]
                 vals.update({
                     "atm_check": check_number,
                     "payment_type": typeofMove, #outbound
                     "payment_method_id": payment_method_id.id,
                     "amount": amount, #cantidad
                     "currency_id": res_company_id.currency_id.id, #outbound
-                    "journal_id": res_company_id.journal_ATM_id.id, #Cash
+                    "journal_id": user_id.journal_ATM_id.id, #Cash
```
Comment (dalonsod) [inline account_invoice_mgmt_atm/controllers/controller.py:101]: We should fire an error if no journal is found for this user

**Review by dalonsod — CHANGES_REQUESTED** [review]
See comments

**Comment by ChristianSantamaria** [comment]
Done :+1:

**Comment by ChristianSantamaria** [comment]
Miss add groups... now its ready

**Pull request merged**